### ISSUE TITLE
ez_setup issues

### ISSUE THREAD

**Issue by MelchiSalins** [SumoLogic/sumologic-python-sdk#6]
Is ez_setup really needed? pip installs break because of it. Here is traceback of it

```
$ pip install sumologic-sdk
You are using pip version 7.1.0, however version 8.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting sumologic-sdk
  Using cached sumologic-sdk-0.1.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/private/var/folders/jp/djdh4dx56011p3hf0m71p2306sp5p0/T/pip-build-o342kT/sumologic-sdk/setup.py", line 1, in <module>
        import ez_setup
    ImportError: No module named ez_setup

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/jp/djdh4dx56011p3hf0m71p2306sp5p0/T/pip-build-o342kT/sumologic-sdk
```

I will submit a patch fixing this. I don't really think using ez_setup is needed if we can just specify that setuptools is required to be installed as a prerequisite.

**Comment by mariten** [comment]
Hi @yowayb @sumoway 
It looks like this user @MelchiSalins fixed this issue about a month ago, and there is a new version `0.1.4`, but that version is still not available on pip.

```
Collecting sumologic-sdk==0.1.4 (from -r requirements.txt (line 10))
  Could not find a version that satisfies the requirement sumologic-sdk==0.1.4 (from -r requirements.txt (line 10)) (from versions: 0.1, 0.1.1, 0.1.2, 0.1.3)
No matching distribution found for sumologic-sdk==0.1.4 (from -r requirements.txt (line 10))
```

Can you publish it?

**Comment by MelchiSalins** [comment]
Hi @mariten 
I did submit a patch for this and as a follow up step I volunteered to publish the artefact but we hit a snag for details see https://github.com/SumoLogic/sumologic-python-sdk/pull/7 . 

TLDR version: 0.1.4 was published by mistake in the past and pypi doesn't allow artefact to be republished with the same version number. So the next release we publish should be > 0.1.4 for pypi to accept it.

I thought it would make more sense to push out a release once we have more bug fixes or features merged into the master branch.

**Comment by mariten** [comment]
Ah I see -- that's unlucky.  Thanks for explaining.
Sumo is recommending people use this SDK in their official seminars so, if it's stable, might not hurt to bump the version up to 1.0.0 soon.  Thanks for creating this library :+1:

**Comment by bbayles** [comment]
Adding my voice here - it would be great to get a new release tagged and pushed to PyPI. As-is I have to pull the tag and install from GitHub, which is awkward!

**Comment by negz** [comment]
Ditto. I was just hit by this bug and would appreciate a fix released.

**Comment by duchatran** [comment]
@negz @bbayles : just fixed and pushed as 0.1.5.  Let me know if it works for you guys.

**Comment by bbayles** [comment]
Thanks for releasing! This version now breaks Python 3 compatibility, I think - it [imports cookielib](https://github.com/SumoLogic/sumologic-python-sdk/blob/master/sumologic/sumologic.py#L5). I'll make a PR to fix that - hopefully you can merge and release that to PyPI?
